library.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://raw.githubusercontent.com/platformio/platformio-core/develop/platformio/assets/schema/library.json",
+  "name": "Libzmanim",
+  "version": "1.0-1",
+  "description": "a C library for zmanim and hebrew dates",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/yparitcher/libzmanim"
+  },
+  "build": {     
+    "unflags":"-std=gnu11",
+    "flags":[
+       "-I include"   
+      ,"-I windows"
+      ,"-D _WIN32"
```

Comment:
There should be no need to define windows.

Reply:
this is one of the hardest things i had here, i wouldn't call it defining windows as more like for platorms who don't have stpncpy , if i won't include the stpncpy.h and the source the file the avr-gcc will fail cause it won't find them, i don't know of any other way to handle this.